training_record/serializers.py

```diff
@@ -62,14 +62,30 @@ class ReadOnlyRecordSerializer(serializers.ModelSerializer):
     role = serializers.IntegerField(
         source='event_coefficient.role',
         read_only=True)
+    allow_ordinary_user_review = serializers.BooleanField(read_only=True,
```

Comment:
this field should indicate whether user can edit the record, similar for other fields

Reply:
‘allow_actions_from_user’